web/impact/impact/tests/test_graphql.py

```diff
@@ -97,11 +99,43 @@ def test_query_with_expert(self):
                     'availableOfficeHours': False,
                     'officeHoursUrl': None,
                     'programInterests': [],
+                    'mentorProgramRoleGrants': {'results': []}
                 }
             }
         }
         self._assert_response_equals_json(query, expected_json)
 
+    def test_mentor_program_role_grants(self):
```

Comment:
can we include another test to confirm that `mentorProgramRoleGrants` will only return program role grants where user role is ["Mentor", "Desired Mentor", "Deferred Mentor"] even if the user has other user roles